libs/dao/src/main/java/com/akto/dao/SingleTypeInfoDao.java

```diff
@@ -52,8 +52,8 @@ public void createIndicesIfAbsent() {
         String[] fieldNames = {SingleTypeInfo._URL, SingleTypeInfo._METHOD, SingleTypeInfo._RESPONSE_CODE, SingleTypeInfo._IS_HEADER, SingleTypeInfo._PARAM, SingleTypeInfo.SUB_TYPE, SingleTypeInfo._API_COLLECTION_ID};
```

Comment:
are we not removing the response code index we discussed

Reply:
I was told by Ankush not to remove any existing indices for now.